README.md

```diff
@@ -84,6 +89,27 @@ while true; do kubectl --namespace tika-test port-forward $POD_NAME 9998:$CONTAI
 * Install it:
   - with Helm 3: `helm install tika . --set image.tag=latest-full`
 
+### Custom configuration for tika
+
+To use custom [configuration](https://tika.apache.org/1.9/configuring.html) values for apache tika, use the `tikaConfig` key in the `values.yaml`.
```

Comment:
The only suggestion I have is to point to more recent documentation.
https://tika.apache.org/2.9.1/configuring.html

Reply:
Thanks for the review. I've updated it.